.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,61 +1,51 @@
 ---
 name: 🐛 Bug report
 about: Create a report to help us improve
-
 ---
+## Bug report
+<!--- Verify first that your issue is not already reported on GitHub -->
+<!--- Also test if the latest release and devel branch are affected too -->
+<!--- This form is processed by Ansibot (machine), complete *all* sections as described -->
 
-<!---
-Verify first that your issue/request is not already reported on GitHub.
-THIS FORM WILL BE READ BY A MACHINE, COMPLETE ALL SECTIONS AS DESCRIBED.
-Also test if the latest release, and devel branch are affected too.
-ALWAYS add information AFTER (OUTSIDE) these html comments.
-Otherwise it may end up being automatically closed by our bot. -->
-
-##### SUMMARY
-<!--- Explain the problem briefly -->
+### Summary
+<!--- Explain the problem briefly below -->
 
-##### ISSUE TYPE
- - Bug Report
+### Component name
+<!--- Write the name of the module, plugin, task or feature below -->
 
-##### COMPONENT NAME
-<!--- Insert, BELOW THIS COMMENT, the name of the module, plugin, task or feature.
-Do not include extra details here, e.g. "vyos_command" not "the network module vyos_command" or the full path-->
+### Ansible version
+<!--- Paste verbatim output from "ansible --version" between quotes -->
+```paste below
 
-##### ANSIBLE VERSION
-<!--- Paste, BELOW THIS COMMENT, verbatim output from "ansible --version" between quotes below -->
 ```
 
-```
+### Configuration
+<!--- Paste verbatim output from "ansible-config dump --only-changed" between quotes -->
```

Comment:
Why `--only-changed` here?

Reply:
It was like this for good reason. We only want to see what the user changed, rather than see the whole configuration (which is a lot).